src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1331,6 +1387,13 @@ rpcClient(std::vector<std::string> const& args,
         nRet                    = rpcINTERNAL;
     }
 
+    if (jvRequest.isMember("jsonrpc"))
+    {
+        jvOutput["jsonrpc"] = jvRequest["jsonrpc"];
```

Comment:
I've been thinking the same thing.  I can do this in a [FOLD] cleanup if everyone is happy with the "big picture".